src/App.js

```diff
@@ -1,14 +1,36 @@
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { Login } from "./Components/Auth/Login/login";
 import { Navbar } from "./Components/Navbar/navbar";
+import { PrivateRoute } from "./Components/PrivateRoute/privateRoute";
+import { LikedVideo } from "./Pages/Liked-Videos/likeVideo";
+import { SingleVideo } from "./Pages/Single-Video-Page/single-video";
 import { VideoListing } from "./Pages/Video-Listing/videoListing";
+import { WatchLater } from "./Pages/WatchLater/watchLater";
```

Comment:
You can use barrel export to avoid importing different components from a single location.